### DISCUSSION TITLE
Is it possible to use microscope in lumen?

### DISCUSSION THREAD

**Discussion by mrmmg** [imanghafoori1/laravel-microscope#197] [Q&A]
Hi
I'm interested with this package and in our company we use lumen as our backend framework (I don't know why we use it, we just use it!). So I want know is it possible to install this package in lumen or not? if not why? which part of code cause that we cannot use it in lumen?

@imanghafoori1 @ahosker @GrahamCampbell @fridzema and others...

**Comment by imanghafoori1** [comment]
I have no idea about the Lumen, you can just test it out.
